txp/Pos/Txp/MemState/Class.hs

```diff
@@ -91,13 +98,15 @@ modifyTxpLocalData f =
         STM.writeTVar txpTip newTip
         STM.writeTVar txpExtra newExtra
         pure res
+ where
+   withLock = L.bracket_ (takeMVar txpLocalDataLock) (putMVar txpLocalDataLock ())
```

Comment:
We already have two ways of doing `bracket`: `MonadMask` and `Mockable Bracket` (which is bad and we are planning to leave only one). It doesn't make sense to add the third one.